### PR TITLE
[WC-1948] Update Cypress  12.16 and Mendix version 10

### PR DESCRIPTION
**Pull request by leonardomendix**
### Pull request type

Test related change (New E2E test, test automation, etc.)

---

### Description

Update Cypress  12.16 and Mendix version 10.

### What should be covered while testing?

Automation should pass.

### CODE REVIEW

**Review by iobuhov — DISMISSED** [review]

File: automation/run-e2e/docker/mxbuild.Dockerfile [hunk 1]
```diff
@@ -20,9 +20,9 @@ RUN \
     apt-get clean && \
 \
     echo "#!/bin/bash -x" >/bin/mxbuild && \
-    echo "mono /tmp/mxbuild/modeler/mxbuild.exe --java-home=/usr/lib/jvm/jdk-11.0.2 --java-exe-path=/usr/lib/jvm/jdk-11.0.2/bin/java \$@" >>/bin/mxbuild && \
+    echo "/tmp/mxbuild/modeler/mxbuild --java-home=/usr/lib/jvm/jdk-11.0.2 --java-exe-path=/usr/lib/jvm/jdk-11.0.2/bin/java \$@" >>/bin/mxbuild && \
```
Comment (iobuhov) [inline automation/run-e2e/docker/mxbuild.Dockerfile:23]: Maybe we should use suggestion from Fabian  and use `dll` extension?

Comment (leonardomendix) [reply automation/run-e2e/docker/mxbuild.Dockerfile]: Oh! Good point!

Done! ✅

File: automation/run-e2e/mendix-versions.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-    "latest": "9.24.0.2965",
-    "8": "8.18.23.62193"
+    "latest": "10.0.0.9976",
```
Comment (iobuhov) [inline automation/run-e2e/mendix-versions.json:2]: nit: let's add alias for 9?

**Review by iobuhov — APPROVED** [review]
Great! LGTM 👍

**Review by samuelreichert — DISMISSED** [review]

File: automation/run-e2e/mendix-versions.json (hunk 2 above)
Comment (leonardomendix) [reply automation/run-e2e/mendix-versions.json:2]: Great! Done! ✅

**Review by iobuhov — APPROVED** [review]

**Review by samuelreichert — APPROVED** [review]

**Pull request merged**